towhee/dataframe/dataframe.py

```diff
@@ -77,29 +88,35 @@ def sealed(self) -> bool:
         """
         return self._sealed
 
+    def _set_frame(self, item):
```

Comment:
so caching and reordering will need to be handled at this step of insert. In addition to, its safe to assume that if the first value added already has a rowid, then every added value will have a row_id. Will this ever change in the future?